day-1/ChronalCalibration/ChronalCalibration/Program.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace ChronalCalibration
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var inputFilePath = @"C:\Users\joakim\repos\AoC-2018\day-1\ChronalCalibration\ChronalCalibration\Input.txt";
+            var numbersAsStringArray = ParseInput(inputFilePath);
+            var numbersAsIntArray = ToIntArray(numbersAsStringArray);
+            var Answer2 = FirstNumberTwice(numbersAsIntArray);
+            var Answer = AddisionOnIntArray(numbersAsIntArray);
+
+            Console.WriteLine("Answer to problem a: " + Answer);
+            Console.WriteLine("Answer to problem b: " + Answer2);
+            Console.ReadKey();
+        }
+
+
+        private static int FirstNumberTwice(int[] intArray)
+        {
+            HashSet<int> myList = new HashSet<int>();
+            var last = 0;
+            myList.Add(0);
+            var sum = 0;
+            while (true)
```

Comment:
Looks like we could get stuck here forever! Should we perhaps define an upper limit for the number of iterations we do on the input data set?